src/content/app/genome-browser/components/browser-sidebar-modal/BrowserSidebarModal.tsx

```diff
@@ -56,12 +50,13 @@ export const browserSidebarModalTitles: { [key: string]: string } = {
   [BrowserSidebarModalView.BOOKMARKS]: 'Previously viewed',
   [BrowserSidebarModalView.SHARE]: 'Share',
   [BrowserSidebarModalView.DOWNLOADS]: 'Downloads',
-  [BrowserSidebarModalView.NAVIGATE_REGION]: 'Change location',
-  [BrowserSidebarModalView.NAVIGATE_LOCATION]: 'Change location'
+  [BrowserSidebarModalView.NAVIGATE]: 'Change location'
 };
 
 export const BrowserSidebarModal = () => {
-  const browserSidebarModalView = useAppSelector(getBrowserSidebarModalView);
+  const browserSidebarModalView = useAppSelector(
+    getBrowserSidebarModalView
+  ) as string;
```

Comment:
This change contradicts line 62 where we are checking `if (!browserSidebarModalView)`. Why are you telling typescript that it must be a string here?

Reply:
Typescript was complaining while working on it. But not anymore. Will change it back.